example_maslow_config.yaml

```
@@ -0,0 +1,49 @@
+# Example configuration showing belt length synchronization settings
+
+kinematics:
+  MaslowKinematics:
+    # Frame anchor coordinates (in mm)
+    tlX: -27.6      # Top left X coordinate
+    tlY: 2064.9     # Top left Y coordinate  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)